datapipelines/src/test/java/io/customer/datapipelines/utils/TestRunPlugin.kt

```diff
@@ -0,0 +1,61 @@
+package io.customer.datapipelines.utils
+
+import com.segment.analytics.kotlin.core.AliasEvent
+import com.segment.analytics.kotlin.core.Analytics
+import com.segment.analytics.kotlin.core.BaseEvent
+import com.segment.analytics.kotlin.core.GroupEvent
+import com.segment.analytics.kotlin.core.IdentifyEvent
+import com.segment.analytics.kotlin.core.ScreenEvent
+import com.segment.analytics.kotlin.core.TrackEvent
+import com.segment.analytics.kotlin.core.platform.EventPlugin
+import com.segment.analytics.kotlin.core.platform.Plugin
+
+class TestRunPlugin(var closure: (BaseEvent?) -> Unit) : EventPlugin {
```

Comment:
Can you please add some comments to explain purpose of this class, would help anyone reading the code later.

Reply:
I can add comment, but I would have hoped the name of the class and code would be enough. As its just recording if a plugin in run.

Reply:
So is this for testing plugin calls? Would appreciate some use cases to understand this better.